README.md

```diff
@@ -2,179 +2,112 @@
+- for Windows, see [the Windows Install Guide](/docs/INSTALL-GUIDE-WINDOWS.md)
+- for Linux, see [the Linux Install Guide](/docs/INSTALL-GUIDE-LINUX.md)
 
-### 2. install gtp2ogs using npm
-
-- For linux in terminal:
-
-```Shell
-sudo npm install -g gtp2ogs
-```
-
-default path install is:
-> /usr/lib/node_modules/gtp2ogs/
-
-- For windows, open a node.js command prompt as admin, then run this command:
-
-```Shell
-npm install -g gtp2ogs
-```
-
-default path install is something like this:
-> C:\Users\yourusername\AppData\Roaming\npm\node_modules\gtp2ogs\
-
-On all operating systems, gtp2ogs will be installed in 2 different directories, but
-**the one that needs to be run with node is gtp2ogs.js in node_modules directory**
-
-### 3. Recommended: Upgrade to devel branch
-
-This step is **highly recommended**
-
-By default, npm installs an old branch of gtp2ogs that does not include latest
-improvements, new features, and fixes
-
-An easy way to upgrade is to copy all the devel gtp2ogs files and folders
-(bot.js, config.js, etc..) to the original directory where gtp2ogs is
-installed, and overwrite the old existing files
-
-note: before overwriting, you can backup your old files so that you can
-go back to the old branch of gtp2ogs anytime later if you want
-
-Then it is needed to do the post install:
-
-The command below will automatically detect all missing packages needed
-from the package.json of the new branch, and install all these packages
-
-- for linux:
-
-```Shell
-cd /usr/lib/node_modules/gtp2ogs/
-sudo npm install
-```
-
-- for windows:
-
-Open a node.js command prompt as admin, then:
+## Options
 
-```Shell
-pushd C:\Users\yourwindowsusername\AppData\Roaming\npm\node_modules\gtp2ogs\
-npm install
-```
+### Options-List
 
-For details or help, you can see:
+You can find the full list of all existing gtp2ogs options [here](/docs/OPTIONS-LIST.md).
 
-- for linux: [3A3) Recommended: Upgrade gtp2ogs.js from old branch to “devel” branch (latest)](https://github.com/wonderingabout/gtp2ogs-tutorial/blob/master/docs/3A3-linux-optional-upgrade-to-devel.md)
-- for windows: [3B3) Recommended: Upgrade gtp2ogs from old branch to devel (latest) branch](https://github.com/wonderingabout/gtp2ogs-tutorial/blob/master/docs/3B3-windows-optional-upgrade-to-devel.md)
+### Options settings
 
-### 4. Most common usage: start gtp2ogs.js using nodejs
+Before putting `options`, you have to first put the mandatory options:
 
-For linux (preferably as sudo):
+- [--username](/docs/OPTIONS-LIST.md#username)
+- [--apikey](/docs/OPTIONS-LIST.md#apikey)
 
-```Shell
-node /usr/lib/node_modules/gtp2ogs/gtp2ogs.js --username <yourbotusername> --apikey <apikey> <gtp2ogsargument1> <gtp2ogsargument2> -- /replace/with/full/path/to/your/ai/runfile.file <botargument1> <botargument2>
-```
+Then, choose the wanted `options` from the above list of all options section
+ (no need to use all these options, only the ones you want), and separate each option with a space.
```

Comment:
It might be better to explain options can be set with command line arguments, and those have to be separated by spaces. This explanation could also come earlier in the introduction i think.

I'm not even sure you have to mention argument have to be space separated, this is pretty standard. Just mention arguments can be used to set the various options.

Reply:
exactly what i did!

then i'll also remove it, it is obvious from the examples anyways so better not overload the readme